### ISSUE TITLE
Sanity init exits early on windows with node 10.5

### ISSUE THREAD

**Issue by bjoerge** [sanity-io/sanity#915]
Got a report about this yesterday, and posting this as an issue for others that may run into it.

Here's an example output:
```
PS C:\Users\IEUser> sanity init
This utility walks you through creating a Sanity installation.
Press ^C at any time to quit.

Looks like you already have a Sanity-account. Sweet!

? Select project to use Create new project
? Informal name for your project Testing PowerShell on windows
? Name of your first dataset: production
? Dataset visibility Public (world readable)
? Output path: C:\Users\IEUser\testingpowershellonwindows
? Select project template (Use arrow keys)
> Movie database (schema + sample data)
  E-commerce (schema + sample data)
  Blog (schema)
  Clean project with no predefined schemas

PS C:\Users\IEUser>
```

Debug info
```
PS C:\Users\IEUser> node -v
v10.5.0
PS C:\Users\IEUser> npm -v
6.1.0
PS C:\Users\IEUser> sanity -v
@sanity/cli version 0.132.5
```

**Comment by bjoerge** [comment]
Doesn't seem to be an issue with Node 10.7.

**Comment by bjoerge** [comment]
Fixed in #927
